cling/bytecode/arrayhist.C

```diff
@@ -2,16 +2,14 @@
 gROOT->Reset();
 char name[4][50];
 
-for(int i=0; i<4; i++)
```

Comment:
Can you open a JIRA ticket to remind us that this syntax (and the one without any curly) is not yet supported and might be supported with additional clang feature?